Sources/ArcGISToolkit/Components/FloorFilter/SiteAndFacilitySelector.swift

```diff
@@ -130,14 +131,19 @@ struct SiteAndFacilitySelector: View {
                         facilities: site.facilities,
                         isHidden: isHidden
                     )
-                    .onDisappear {
-                        shouldAutoSelect = false
-                    }
```

Comment:
If a site was deselected as a result of a map pan, the `NavigationLink` would close causing the computed binding above to return `nil`, producing the jumpy behavior.